packages/cfd/src/Containers/__tests__/cfd-password-modal.spec.js

```diff
@@ -0,0 +1,513 @@
+            </Router>
+        );
+
+        expect(await screen.findByTestId('IcMt5CfdPlatform')).toBeInTheDocument();
+    });
+
+    it('should invoke verifyEmail for DerivX when Forgot password is clicked', async () => {
+        const mockDisableCFDPasswordModalFn = jest.fn();
+        const props = {
+            ...mock_props,
+            is_cfd_password_modal_enabled: true,
+            is_cfd_success_dialog_enabled: false,
+            account_status: { status: [], category: 'Real' },
+            getAccountStatus: mockFn,
+            platform: 'dxtrade',
+            error_type: 'PasswordReset',
+            setCFDSuccessDialog: mockFn,
+            setMt5Error: mockDisableCFDPasswordModalFn,
+            disableCFDPasswordModal: mockFn,
+            account_type: { category: 'demo', type: 'financial' },
+            email: 'demo@deriv.com',
+        };
+
+        render(
+            <Router history={history}>
+                <CFDPasswordModal {...props} />
+            </Router>
+        );
+        const ele_cancel_btn = await screen.findByRole('button', { name: /Forgot password?/i });
+        fireEvent.click(ele_cancel_btn);
+        await waitFor(() => {
+            expect(WS.verifyEmail).toHaveBeenCalledWith('demo@deriv.com', 'trading_platform_dxtrade_password_reset', {
+                url_parameters: { redirect_to: 21 },
+            });
+        });
+    });
+
+    it('should create DMT5 password when clicked on Create DMT5 password', async () => {
+        const user_input = 'zo8lAet#2q01Ih';
+        validPassword.mockReturnValue(true);
+        const props = {
+            ...mock_props,
+            is_cfd_password_modal_enabled: true,
+            is_cfd_success_dialog_enabled: false,
+            account_status: { status: ['mt5_password_not_set'], category: 'Real' },
+            getAccountStatus: mockFn,
+            platform: 'mt5',
+            account_type: { category: 'real', type: 'financial' },
+            submitMt5Password: mockFn,
+        };
+
+        render(
+            <Router history={history}>
+                <CFDPasswordModal {...props} />;
+            </Router>
+        );
+        fireEvent.change(await screen.findByTestId('mt5_password'), { target: { value: user_input } });
+        fireEvent.click(await screen.findByRole('button', { name: 'Create DMT5 password' }));
+
+        expect(await screen.findByTestId('dt_initial_loader')).toBeInTheDocument();
+    });
+
+    it('should create CFD password when clicked on Add account', async () => {
+        const user_input = 'zo8lAet#2q01Ih';
+        const mockSubmitCFDPasswordFn = jest.fn();
+        validPassword.mockReturnValue(true);
+        const props = {
+            ...mock_props,
+            is_cfd_password_modal_enabled: true,
+            is_cfd_success_dialog_enabled: false,
+            account_status: { status: ['mt5_password_not_set'], category: 'Real' },
+            getAccountStatus: mockFn,
+            platform: 'dxtrader',
```

Comment:
Here the platform is "dxtrade". So can we change the description to "should create Dxtrade password when clicked on Add account"

Reply:
Will rename